recipes/qt/5.x.x/conanfile.py

```diff
@@ -0,0 +1,710 @@
+import os
+import shutil
+import itertools
+import glob
+
+import configparser
+from conans import ConanFile, tools, __version__ as conan_version, RunEnvironment
+from conans.errors import ConanInvalidConfiguration
+from conans.model import Generator
+from conans.tools import Version
+
+
+class qt(Generator):
+    @property
+    def filename(self):
+        return "qt.conf"
+
+    @property
+    def content(self):
+        return "[Paths]\nPrefix = %s\n" % self.conanfile.deps_cpp_info["qt"].rootpath.replace("\\", "/")
+
+
+class QtConan(ConanFile):
+    _submodules = ["qtsvg", "qtdeclarative", "qtactiveqt", "qtscript", "qtmultimedia", "qttools", "qtxmlpatterns", 
+    "qttranslations", "qtdoc", "qtrepotools", "qtqa", "qtlocation", "qtsensors", "qtconnectivity", "qtwayland",
+    "qt3d", "qtimageformats", "qtgraphicaleffects", "qtquickcontrols", "qtserialbus", "qtserialport", "qtx11extras",
+    "qtmacextras", "qtwinextras", "qtandroidextras", "qtwebsockets", "qtwebchannel", "qtwebengine", "qtwebview", 
+    "qtquickcontrols2", "qtpurchasing", "qtcharts", "qtdatavis3d", "qtvirtualkeyboard", "qtgamepad", "qtscxml",
+    "qtspeech", "qtnetworkauth", "qtremoteobjects", "qtwebglplugin", "qtlottie", "qtquicktimeline", "qtquick3d"]
```

Comment:
I love to see this list in a separate file god forbid it changes

https://github.com/conan-io/conan-center-index/blob/b011e7bbab6790f7fdc11aadc7140343b99a11fe/recipes/open62541/all/conanfile.py#L167-L168

Reply:
this feels overkill, for a simple list